### PR TITLE
Added Security disclousure policy

### PR DESCRIPTION
**Pull request by DicsyDel**
Resolves #

## Target Release

1.6.0

### NEW FEATURES | UPGRADE NOTES | ENHANCEMENTS | BUG FIXES | EXPERIMENTS

 

-

### CODE REVIEW

**Comment by DicsyDel** [comment]
Fixes: https://github.com/opentofu/opentofu/issues/423

File: README.md
Comment (brikis98) [inline README.md:34]: Should this not link to `SECURITY.md` instead of repeating _some_ of that content here?

File: SECURITY.md
Comment (brikis98) [inline SECURITY.md:3]: Do we really want crash reports at this email alias? Seems like crash reports should be filed as bugs in the repo, and only vulnerability disclosures should go to the email address?

File: SECURITY.md
Comment (brikis98) [inline SECURITY.md:1]: Once this PR is merged, we should add a link to these docs to the opentofu website.

File: SECURITY.md
Comment (brikis98) [inline SECURITY.md:23]: No one outside this group can open this link. I'd remove it.

File: SECURITY.md
Comment (brikis98) [inline SECURITY.md:68]: To ensure an SLA of 24h, we'd have to have on-call rotations and a bunch of other stuff that sounds somewhat impractical for an open source foundation? At least, until we have a fully paid team working on this?

**Comment by allofthesepeople** [comment]
Should we add a section supported versions? e.g. 2 minor versions?

File: SECURITY.md
Comment (brikis98) [inline SECURITY.md:100]: Did we set up this list yet? Who is on it? Do we plan to set up an on call rotation to respond to these issues within these SLAs/SLOs?

**Comment by allofthesepeople** [comment]
Further: should we state where to follow security updates? Mailing list, blog posts etc?

File: SECURITY.md
Comment (allofthesepeople) [inline SECURITY.md:103]: should this be the "PST" rather than "OpenToFu security team"?

File: SECURITY.md
Comment (DicsyDel) [reply SECURITY.md]: I've updated it to 72 hours. I think it's reasonable across the entire Core team and SC members. We can always update it in the future. It's handled via a mailing list, so I don't think we should have an on-call rotations for this matter (at least right now).

File: SECURITY.md
Comment (DicsyDel) [reply SECURITY.md:99]: It's a mailing list that consists of security teams from different vendors who are part SC. It's already configured. No plans for on-call rotation for now. We can change this later if there will be a need or issues with current process.

**Comment by DicsyDel** [comment]
> Should we add a section supported versions? e.g. 2 minor versions?

I don't think it's necessary right now, as we don't even have a stable release yet. We can add it in the future. Trying to keep scope small.

File: SECURITY.md
Comment (brikis98) [reply SECURITY.md]: I guess I'm thinking primarily of the holidays: e.g., a security report comes in during Christmas... Are we sure someone will be around to respond promptly?

Comment (brikis98) [reply SECURITY.md]: I guess to put it another way: do we want to commit, publicly and in writing, to specific SLAs/SLOs at this stage in the project? We will certainly react to security issues as quickly as we possibly can, but do we want to make that a hard commitment at this stage, before full-time, dedicated people are even hired?

**Review by brikis98 — APPROVED** [review]
Overall, LGTM. Thanks for putting this together 👍 

My only concern is if we want to commit to SLAs/SLOs right now. I know we'll act quick on all security issues, but as we haven't hired a dedicated, full-time team for this, I worry about making promises like this.

File: SECURITY.md
Comment (DicsyDel) [reply SECURITY.md]: @brikis98 I agree, you right. I've removed sections with SLO / timelines for now.

**Pull request merged**
